test/integration/acceptance/grantable_permissions_fixture.cpp

```diff
@@ -0,0 +1,222 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "integration/acceptance/grantable_permissions_fixture.hpp"
+
+using namespace shared_model::interface::permissions;
+
+shared_model::proto::Transaction
+GrantablePermissionsFixture::makeAccountWithPerms(
+    const shared_model::interface::types::AccountNameType &user,
+    const shared_model::crypto::Keypair &key,
+    const shared_model::interface::RolePermissionSet &perms,
+    const shared_model::interface::types::RoleIdType &role) {
+  return createUserWithPerms(user, key.publicKey(), role, perms)
+      .build()
+      .signAndAddSignature(kAdminKeypair)
+      .finish();
+}
+
+integration_framework::IntegrationTestFramework &
+GrantablePermissionsFixture::createTwoAccounts(
+    integration_framework::IntegrationTestFramework &itf,
+    const shared_model::interface::RolePermissionSet &perm1,
+    const shared_model::interface::RolePermissionSet &perm2) {
+  itf.setInitialState(kAdminKeypair)
```

Comment:
Won't it cause issues if initial state is already set?

Reply:
Will move setInitialState from createTwoAccounts method.